Melonie's Solution/DB Challenge/dataMigrate/migrate.java

```diff
@@ -0,0 +1,80 @@
+package dataInput;
+import java.io.IOException;
+import java.io.*;
+import java.util.Properties;
+
+public class migrate { 
+
+	     // export
+	     public static void exportSql() throws IOException{
+	         Properties properties = new Properties();  
+	         //  read the properties file
+	         properties.load(Import.class.getClassLoader().getResourceAsStream("jdbc.properties")); 
+	         Runtime runtime = Runtime.getRuntime();  
+	         String command = getExportCommand(properties);  
+	         
+	         runtime.exec(command)�� 

```

Comment:
Could you give a bit more insight on your decision to migrate the data using shell command execution? I noticed you had JDBC on all of your property names; did you initially start with JDBC then changed your approach afterwards?